template.rb

```diff
@@ -69,14 +69,23 @@
   gem 'truncate_html'
   gem 'view_component', '~> 2.46'
 
+  gem_group :test do
+    # Include the system tests gem generated by Rails on new installs with
+    # minitest, since many apps will be generated passing `--skip-test` we
+    # need to support both scenarios.
```

Comment:
Can you elaborate on this comment? It mentions several things (system tests gem, minitest, `skip-test`, "both scenarios"), but I don't understand how they relate with capybara and the other test gems below.

Reply:
I'm replacing it with:

```
    # We need to add capybara along with a javascript driver to support the provided system specs,
    # `rails new` will add the following gems for system tests unless `--skip-test` is provided.
    # We want to stick with them but we can't be sure about how the app was generated, so we'll
    # add them only if they're not already in the Gemfile.
```

hope it makes more sense, but basically you'll get those three gems in your pristine `rails new` gemfile depending on which flags you provided. We also want them as they're the default for rails testing but we need to take into account that they might be already there.